### DISCUSSION TITLE
framework.ini doesn't save custom resolution

### DISCUSSION THREAD

**Discussion by nyaruku** [ppy/osu#31242] [Q&A]
### Type

Other

### Bug description

i tried to change my fullscreen resolution to 1440x1080 over framework.ini located in the roaming appdata folder of osu! lazer and once i launch lazer, it doesn't apply the resolution i've set in the file...
(yes i need that resolution, bc on a 1920x1080 1080p screen, the height is 1080 and so i dont want that pixels are interpolated, results in blurry view)

### Version

2024.1219.2-lazer

### Logs

None

**Comment by peppy** [comment]
Please show me your display adaptor resolution list with 1440x1080 displayed.

We can only set resolutions which are officially supported by your graphics drivers so this is important.

**Reply by nyaruku** [reply]
> Please show me your display adaptor resolution list with 1440x1080 displayed.
> 
> We can only set resolutions which are officially supported by your graphics drivers so this is important.

![image](https://github.com/user-attachments/assets/fa9a5d4c-66d1-42ed-ba4d-1e68d20597d8)

**Reply by peppy** [reply]
Have you added this yourself via tooling? It doesn't sound standard to me.

**Reply by nyaruku** [reply]
> Have you added this yourself via tooling? It doesn't sound standard to me.

yes, i added it through the Nvidia control panel, but even then, i think this resolution should be added to have a non blurry 4:3 aspect ratio on 1920x1080(1080p) screens, as the height matches the native height, resulting in a non blurry/interpolated view
